tools/tf2openapi/types/tfmetagraph.go

```diff
@@ -6,31 +6,33 @@ It is the internal model representation for the MetaGraph defined in the TensorF
 [tensorflow/core/protobuf/meta_graph.proto]
 */
 import (
+	"fmt"
 	"github.com/getkin/kin-openapi/openapi3"
 	pb "github.com/kubeflow/kfserving/tools/tf2openapi/generated/protobuf"
 )
 
-const PredictReqSigDefMethod string = "tensorflow/serving/predict"
-
 type TFMetaGraph struct {
 	SignatureDefs [] TFSignatureDef
 }
 
-func NewTFMetaGraph(metaGraph *pb.MetaGraphDef) (TFMetaGraph, error) {
+func NewTFMetaGraph(metaGraph *pb.MetaGraphDef, sigDefKey string) (TFMetaGraph, error) {
 	tfMetaGraph := TFMetaGraph{
 		SignatureDefs: []TFSignatureDef{},
 	}
 	for key, definition := range metaGraph.SignatureDef {
```

Comment:
`metaGraph.SignatureDef` is a map right? You should be able to search for the `sigDefKey` directly.

Reply:
Yes, it's a map. Got it.